packages/experimental/src.ts/nonce-manager.ts

```diff
@@ -63,16 +53,6 @@ export class NonceManager extends ethers.Signer {
     }
 
     sendTransaction(transaction: ethers.utils.Deferrable<ethers.providers.TransactionRequest>): Promise<ethers.providers.TransactionResponse> {
```

Comment:
I think that nonce-manager will not be needed in Hedera case. We should add todo at the start of the nonce-manager file to address whether we need it at all